ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -610,20 +620,11 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
     const isEmpty = (val: string) => val == null || util.isEmpty(val.trim());
     const format = (val: string) => isEmpty(val) ? '' : ', ' + val.trim();
     const streetFormat = (val: string) => isEmpty(val) ? '' : val.trim();
+      // tslint:disable-next-line:max-line-length
     return `${isEmpty(a.name) ? '' : a.name}${isEmpty(a.name) ? streetFormat(a.street1) : format(a.street1)}${format(a.street2)}${format(a.city)}${format(a.state)}`
       + `${format(a.zip)}${format(a.country)}${isEmpty(a.phone) ? '' : ', Phone: ' + a.phone}`;
   }
 
-  private lookupErrorMessage(currError: AddressError): string {
-    const CODE_TO_MESSAGE = {
-      'E.HOUSE_NUMBER.INVALID': 'We could not find the street number provided.',
-      'E.ADDRESS.NOT_FOUND': 'We could not find the entered address.',
-      'E.STREET.MAGNET': 'Street address is ambiguous.'
-    };
-    return CODE_TO_MESSAGE[currError.code] ? CODE_TO_MESSAGE[currError.code]
-      : (currError.message.endsWith('.') ? currError.message : currError.message + '.');
-  }
-
```

Comment:
As mentioned earlier, these rewritten error messages just went into the translation json file.